articles/fleet-4.26.0.md

```diff
@@ -0,0 +1,88 @@
+
+## Log user activity for audits
+**Available in Fleet Premium**
+
+Security and IT administrators have long to-do lists and short deadlines. Increasing access to Fleet across the company would help lighten the workload, but more users could mean more chances for things to fall through the cracks. Fleet 4.26.0 gives you extra confidence to extend your user base. 
+
+Now you can stream Fleet user activities to external destinations, aggregating granular data for greater insights in the event issues occur.
+
+To make sure administrative operations run smoothly, Fleet streams activity to log destinations asynchronously. Activity will still appear in the Fleet UI in real time, but streaming this data may take up to 5 minutes.
+
+## Ingest the latest software data
+**Available in Fleet Free and Fleet Premium**
+
+You already have a lot of raw data to sift through in your data lake, especially if your organization has hundreds of thousands of devices. What if you could refine your software data before it reaches the lake?
+
+Fleet 4.26.0 reduces the number of calls you have to make to pull software data with the REST API. Each time a host has software added, updated, or deleted, a `host_software_updated_at` timestamp gets updated for that host. The `host_software_updated_at` timestamp is exposed through the API. This lets you send the latest software data to your data lake, so you can avoid drowning in outdated information.
+
+## Fleet MDM
+**MDM features are not ready for production and are currently in development. These features are disabled by default.**
+
+Fleet is building a cross-platform MDM to give IT and security teams the visibility and openness they need. Here are the latest developments:
+
+- Added functionality to ingest device information from Apple MDM endpoints so that a device ordered in Apple Business Manager can be surfaced in Fleet.
+- Added new activities to the activities API when a device has MDM is turned on or off..
+- Added option to filter hosts by MDM status "pending" to surface devices ordered through Apple Business Manager that are still pending enrollment to Fleet.
+- Added a flag to indicate if the Apple Business Manager terms and conditions have changed and must be accepted to have automatic enrollment of hosts work again. A banner is added to the output of `fleetctl` commands when this is the case.
+- Added side navigation layout to the integration page and conditionally show MDM section.
+- Added a modal to allow users to download an enrollment profile required for turning on MDM.
+- Added a new configuration option to set the default team for Apple Business Manager.
+
+Are you interested in the Fleet MDM beta? [Schedule a call](https://calendly.com/fleetdm/demo) to save your spot.
+
+## More new features, improvements, and bug fixes
+- Added locally-formatted datetime tooltips.
+- Added Pendo to Sandbox instances.
```

Comment:
```suggestion
```

I think we can remove this item. My understanding is that the release blogpost targets Fleet users. This only affects Sandbox instances. Fleet users don't interact with Fleet Sandbox.